demo/odaslive/parameters.c

```diff
@@ -244,6 +288,89 @@
 
     }
 
+    pa_channel_map* parameters_pa_channel_map_config(const char* fileConfig)
+    {
+        bool channelNamesExists = parameters_exists(fileConfig, "raw.interface.channelnames");
+        bool channelMapExists = parameters_exists(fileConfig, "raw.interface.channelmap");
+        if (channelNamesExists && channelMapExists)
+        {
+            printf("Configuration error: raw.interface.channelnames and raw.interface.channelmap cannot be used at the same time\n");
+            exit(EXIT_FAILURE);
+        }
+        else if (!(channelNamesExists || channelMapExists))
+        {
+            printf("Configuration error: either raw.interface.channelnames or raw.interface.channelmap must be used with pulseaudio\n");
+            exit(EXIT_FAILURE);
+        }
```

Comment:
Two modes:
`channelmap = ("front-left", "front-right");` -> Mapping more explicit
`channelnames = "front-left,front-right";` -> Can be directly copied from the output of `pacmd list-sources`

Might be better to only have one way. Which one would be better?

Reply:
I think ("front-left", "front-right"); is better since it's the format for lists.

Reply:
Removed the `channelnames = "front-left,front-right";` version